Set up SAML authentication with Okta (#1675)
timescale/docs: _troubleshooting/mst/okta-password.md

---
title: The Okta password does not work
section: troubleshooting
products: [mst]
topics: [authentication]
errors:
  - language: bash
    message: |- 
keywords: [authentication, SAML, Okta]
tags: [mst]
---

> Make sure to use the `Account Link URL` to add the Okta Authentication method to

Reviewer: ```suggestion
Make sure you use the `Account Link URL` to add the Okta Authentication method to
```

your MST profile.

> After it is linked, you should get the choice of multiple sign-in methods as

Reviewer: ```suggestion
After it is linked, you get a choice of authentication methods, and 
```

> well as see the other Authentication method in `User Information`->

Reviewer: ```suggestion
can see the other authentication method in the`User Information`->
```

> `Authentication` section on the [MST account] [mst-login].

Reviewer: ```suggestion
`Authentication` section in your [MST account] [mst-login].
```
